guides/getting-started.md

```diff
@@ -157,7 +157,20 @@ System.config({
     // CDK individual packages
     '@angular/cdk/platform': 'npm:@angular/cdk/bundles/cdk-platform.umd.js',
     '@angular/cdk/a11y': 'npm:@angular/cdk/bundles/cdk-a11y.umd.js',
+    
+    // hammerjs
+    'hammerjs': 'npm:hammerjs'
     // ...
+  },
+  
+  packages:{
+     app: {
+     
+     },
```

Comment:
What is this `app` objecT? Can it be removed?

Reply:
yes, please